addon/components/frost-link.js

```diff
@@ -258,7 +264,7 @@ export default LinkComponent.extend(PropTypeMixin, HookMixin, SpreadMixin, {
    * @returns {boolean} true if we should open the link in the current tab and false otherwise
    */
   _shouldOpenInSameTab () {
-    return !(this.get('priority') === 'primary' && this.get('disabled') === false)
+    return this.get('priority') !== 'primary'
```

Comment:
There was a bug here. We were able to click even if the link was disabled. I fixed it and added a test.